packages/kbn-esql-validation-autocomplete/src/validation/__tests__/validation.command.stats.test.ts

```diff
@@ -0,0 +1,360 @@
+/*
```

Comment:
I moved all `STATS` command tests to this separate file and grouped them using `describe(` into meaningful groups. The `METRICS` command works very much like the `STATS` command, so I needed to see all these tests to replicate them for `METRICS`.